src/renderers/WebGLRenderer.js

```diff
@@ -2806,8 +2812,6 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	function setTexture2D( texture, slot ) {
 
-		if ( texture instanceof THREE.WebGLRenderTarget ) texture = texture.texture;
```

Comment:
Ah good! That one looks just fine. 

It'd be ideal to use the same scheme for the cube case.